Pull in r198739 from upstream llvm trunk (by Venkatraman Govindaraju):

  [Sparc] Correct the mask for fixup_sparc_br19.

Introduced here: http://svnweb.freebsd.org/changeset/base/262261

Index: lib/Target/Sparc/MCTargetDesc/SparcAsmBackend.cpp
===================================================================
--- lib/Target/Sparc/MCTargetDesc/SparcAsmBackend.cpp
+++ lib/Target/Sparc/MCTargetDesc/SparcAsmBackend.cpp
@@ -31,7 +31,7 @@ static unsigned adjustFixupValue(unsigned Kind, ui
   case Sparc::fixup_sparc_br22:
     return (Value >> 2) & 0x3fffff;
   case Sparc::fixup_sparc_br19:
-    return (Value >> 2) & 0x1ffff;
+    return (Value >> 2) & 0x7ffff;
   case Sparc::fixup_sparc_hi22:
     return (Value >> 10) & 0x3fffff;
   case Sparc::fixup_sparc_lo10:
